feat(changelog): transactional-email-added-tem-image-attachment 2023-02-09 (#1292)
scaleway/docs-content: changelog/february2023/2023-02-09-transactional-email-added-tem-image-attachment.mdx

---

> title: TEM image attachment

Reviewer: ```suggestion
title: Image attachments are now available!
```

status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'

>     url: 'https://scaleway-community.slack.com/join/shared_invite/zt-xqd22ppu-whoKrK9XWizEO85MbsurLA#/shared-invite/emailhttps://slack.scaleway.com'

Reviewer: ```suggestion
    url: 'https://scaleway-community.slack.com'
```

date: 2023-02-09
category: managed-services
product: transactional-email
---

You can now send images as attachments in your transactional emails, supported formats:  jpeg, jpg, png, gif. Click here for the documentation.

> (HI team doc, how can I add the link to the doc: https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/)

Reviewer: ```suggestion
[Find out about all the attachments types available.](https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/)
```

---
title: Image attachments are now available!
status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://scaleway-community.slack.com'
date: 2023-02-09
category: managed-services
product: transactional-email
---

You can now send images as attachments in your transactional emails. The supported formats are the following: 
* `jpeg`
* `jpg`
* `png`

> * `gif`.

Reviewer: ```suggestion
* `gif`
```
